src/main/webapp/WEB-INF/jsp/signup.jsp

```diff
@@ -0,0 +1,117 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+
+<html>
+<head>
+<title>Sign up Page</title>
+
+<link rel="stylesheet" href="/formbuilder/assets/vendors/bootstrap-3.3.7-dist/css/bootstrap.min.css">
+<link rel="stylesheet" href="/formbuilder/assets/vendors/bootstrap-material-design/css/bootstrap-material-design.min.css">
+<link rel="stylesheet" href="/formbuilder/assets/vendors/bootstrap-material-design/css/ripples.min.css">
+<link rel="stylesheet" href="/formbuilder/assets/vendors/DataTables-1.10.13/css/dataTables.bootstrap.css">
+<link rel="stylesheet" href="/formbuilder/assets/resources/css/formbuilder.css">
+
+<script src="/formbuilder/assets/vendors/jquery/jquery-3.2.0.min.js"></script>
+<script src="/formbuilder/assets/vendors/bootstrap-3.3.7-dist/js/bootstrap.min.js"></script>
+
+<!-- Material Design for Bootstrap -->
+<script src="/formbuilder/assets/vendors/bootstrap-material-design/js/material.js"></script>
+<script src="/formbuilder/assets/vendors/bootstrap-material-design/js/ripples.min.js"></script>
+</head>
+<body>
+
+<div class="container">
+
+	<div class="col-md-offset-3 col-md-7">
+		<h3>Thank you for being part of our community</h3>
+		
+		<div class="panel panel-primary">
+		
+			<div class="panel-heading">
+				<h4 class="panel-title">Signup</h4>
+			</div>
+				
+				<form:form modelAttribute="user" class="User" method="post" >
```

Comment:
Formatting for form is off. Please add proper grid styling as documented in grid system for bootstrap.
https://getbootstrap.com/examples/grid/